test_runner/build.gradle.kts

```diff
@@ -258,3 +258,17 @@ tasks.assemble {
     dependsOn(generateManpageAsciiDoc)
 }
 // end --- ASCII doc generation ---
+
+val updateVersion by tasks.registering {
+    shouldRunAfter(tasks.processResources)
+    doLast {
+        File("${project.buildDir}/resources/main/version.txt").writeText("local_snapshot")
+        File("${project.buildDir}/resources/main/revision.txt").writeText(
```

Comment:
The bitrise job does this:

```
echo "$BITRISE_GIT_TAG" > ./test_runner/src/main/resources/version.txt
echo "$GIT_CLONE_COMMIT_HASH" > ./test_runner/src/main/resources/revision.txt
```

and then the gradle commands are run. I think this may break the bitrise release job.

Reply:
Thanks for info. So I will use `BITRISE_IO` env for turning off this task when running on bitrise.